lib/file-info-view.coffee

```diff
@@ -10,10 +12,30 @@ class FileInfoView extends HTMLElement
     @bufferModified.classList.add('buffer-modified')
     @appendChild(@bufferModified)
 
+    @newTooltip()
+
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
+    clickHandler = ->
+      atom.clipboard.write(@getActiveItem()?.getPath?())
+      setTimeout =>
+        @newTooltip()
+      , 2000
+
+    @addEventListener('click', clickHandler)
+    @clickSubscription = new Disposable => @removeEventListener('click', clickHandler)
+
+  newTooltip: ->
```

Comment:
I see your point. So I guess that would result in a fancy `createOrReplaceTooltip` name :frowning: . Well, seen a lot longer names in the atom codebase, to be honest :smiley: . Hopefully we will get feedback from the atom gurus on this topic too.